paclab/urls.py

```diff
@@ -14,12 +14,17 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
+from django.contrib.auth import views as auth_views
 from django.urls import path, include
 from django.conf import settings
 from django.conf.urls.static import static
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('passwordreset/', auth_views.PasswordResetView.as_view(template_name='passwordReset.html'), name='password_reset'),
```

Comment:
lets change the urls for this group. Something like 'password/reset/', 'password/reset/done'/, 'password/reset/confirm/...', 'password/reset/completed/' etc. give the URLs a bit of hierarchy